CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [v1.8.0-dev] - Unreleased
 
 ### Added
+- Support skills in periodic heuristic (`first_solution_strategy='periodic'`) [#194](https://github.com/Mapotempo/optimizer-api/pull/194)
```

Comment:
I believe it was me who suggested that part but I must have copy/pasted the `first_solution_strategy='periodic'` suggestion from the Changelog. If we prefer `:`, should fix both (and others if exists) so that we can prevent this back and forth.